bin/cli-flags.js

```diff
@@ -98,6 +98,22 @@ module.exports = {
         delete opts.clientProgress;
       },
     },
+    {
+      name: 'client-overlay',
+      type: Boolean,
+      configs: [
+        {
+          type: 'boolean',
+        },
+      ],
+      description:
+        'Show a full-screen overlay in the browser when there are compiler errors or warnings.',
+      processor(opts) {
```

Comment:
maybe we can have test to ensure processor behaviour in future

/cc @alexander-akait?

Reply:
Yep, e2e testing with `playwright` or `puppeteer`, let's do it in other PR